app/Platform/src/Providers/PlatformServiceProvider.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Platform\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class PlatformServiceProvider extends ServiceProvider
+{
+	public function register()
```

Comment:
Do we need this file at all for now? We can easily add it later if needed. If you plan to keep it, you can remove register/boot